packages/functions/src/cache/in-memory-cache.ts

```diff
@@ -0,0 +1,67 @@
+import { FunctionCache } from './types';
+
+interface CacheEntry {
+  value: unknown;
+  lastModified: number;
+  tags?: string[];
+  ttl?: number;
```

Comment:
Is `ttl` in seconds? Let's leave a comment to make it clear.

Reply:
updated